src/main/java/gregtech/common/crafting/GTIngredientNBT.java

```diff
@@ -0,0 +1,10 @@
+package gregtech.common.crafting;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.crafting.IngredientNBT;
+
+public class GTIngredientNBT extends IngredientNBT {
+    protected GTIngredientNBT(ItemStack stack) {
+        super(stack);
+    }
+}
```

Comment:
I am still somewhat confused on the purpose for this class. Crafting ingredients are assigned as GTIngredientNBT, but then nothing is done with that assignment. In addition, this class does not have any special overrides, all it does is call super.

Reply:
IngredientNBT constructor is protected